src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -458,6 +458,16 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
       return
     }
 
+    // if the original legal type is one of the continued-in types, alter to another Cont. In type
+    if (
+      this.getOriginalLegalType === CorpTypeCd.BEN_CONTINUE_IN ||
+      this.getOriginalLegalType === CorpTypeCd.CCC_CONTINUE_IN ||
+      this.getOriginalLegalType === CorpTypeCd.CONTINUE_IN ||
+      this.getOriginalLegalType === CorpTypeCd.ULC_CONTINUE_IN
+    ) {
+      this.selectedEntityType = convertToContinuedInEntityType(this.selectedEntityType)
+    }
+
```

Comment:
I'm thinking you can remove `convertToContinuedInEntityType()` ...?

Reply:
See also `convertToRegularEntityType()`.

Ref: https://github.com/bcgov/business-edit-ui/pull/582/files#diff-ca058487aa6730e697018454f9201fad0a0c649b2084211046382fa3e93b6c61

Reply:
Are these 2 methods used in the previous ticket to solve the issue that for Cont in, if the user clicks on "Change" for the Business Type then the Business Type Selection is blank?

Reply:
Yes..